dialogueMaker/dialogueMaker.cpp

```diff
@@ -9,24 +9,23 @@ Scene::Scene(std::string id, std::string dialogue, bool isEndScene) {
 }
 
 void Scene::printScene() {
-  /*for (int i = 0; i < sceneLength; i++) {
-    std::cout << "-";
-  }*/
-  std::cout << "---------------------------------------------------------------------------------------------------------------------------";
-  //std::cout << '\n' << dialogue << '\n';
-  std::cout << '\n';
-  for (int i = 0; i < dialogue.size(); i++)
-  {
-    std::cout << dialogue[i];
-    Sleep(1);
+  // Print ASCII art for the scene
+  std::string asciiArt = getSceneASCII(id);
+  if (!asciiArt.empty()) {
+        std::cout << asciiArt << std::endl;
   }
-  std::cout << '\n';
   std::cout << "---------------------------------------------------------------------------------------------------------------------------";
+  std::cout << '\n' << dialogue << '\n';
+  // std::cout << '\n';
+  // for (int i = 0; i < dialogue.size(); i++)
+  // {
+  //   std::cout << dialogue[i];
+  //   Sleep(1);
+  // }
+  // std::cout << '\n';
+  // std::cout << "---------------------------------------------------------------------------------------------------------------------------";
```

Comment:
So, no more animation for the dialogue?. And what about the bottom border on line 26?

Reply:
sr sir, I forgot to change it back because when testing it took a long time and I will continue styling the code.

Reply:
Oh okay. So you will change it back right?

Reply:
Yes